catalog/app/containers/Bucket/PackageList.js

```diff
@@ -386,7 +391,7 @@ export default function PackageList({
 
   return (
     <>
-      <UploadDialog bucket={bucket} open={uploadOpen} onClose={closeUpload} />
+      <UploadDialog {...{ bucket, refresh, open: uploadOpen, onClose: closeUpload }} />
```

Comment:
I think that names were implied to be "**openUpload**" and "closeUpload"

Reply:
`uploadOpen` is a state, more like `uploadDialogIsOpen`, and `closeUpload` is an action/verb